src/core/manager-serialize.c

```diff
@@ -397,15 +394,6 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                         else
                                 m->service_watchdogs = b;
 
-                } else if ((val = startswith(l, "honor-device-enumeration="))) {
```

Comment:
Wouldn't this cause a warning if an older systemd version passed `honor-device-enumeration` in?

Reply:
Ouch. Thanks.

Reply:
Fixed.